weblate/lang/data.py

```diff
@@ -346,6 +346,7 @@
     'schinese': 'zh_Hans',
     'chinese_zh': 'zh_Hant',
     'tchinese': 'zh_Hant',
+    'korean': 'ko_KR',
```

Comment:
Wouldn't it be better to alias it to `ko` only? The country specification is usually not needed and the alias should be consistent with others which do not use country unless necessary...

Reply:
Will it work like that? I see no definition for "ko" in data.py

Reply:
The language definitions have been recently moved to separate file:

https://github.com/WeblateOrg/weblate/blob/4ded14d535b02c306f1cfaa5268557ef4711e3e5/weblate/langdata/languages.py#L943-L948

Reply:
changed to ko